packages/spark-core/base/inputs/_input-container.scss

```diff
@@ -0,0 +1,21 @@
+.sprk-b-InputContainer {
+  max-width: $input-max-width;
+  margin-top: 2.5rem;
+}
+
+.sprk-b-InputContainer\@sm {
```

Comment:
The format for responsive suffixes is `.thing--what-it-does@breakpoint`. For example, our buttons are `sprk-c-Button--full\@sm`. It's saying the buttons are full at the small breakpoint. I searched this PR for this selector and only saw it here. Maybe its some dev code that you forgot to remove?

Reply:
Its code that was there before (I think), which isnt necessary now, I'll remove it.